src/components/SegmentedControls/SegmentedControls.tsx

```diff
@@ -133,6 +138,7 @@ export const SegmentedControls = ({
     ariaLabel = 'SegmentedControls',
     disabled = false,
     hugWidth = false,
+    padding = 'default',
```

Comment:
I would use the concept of `variant`, instead of `padding`. That way, if we need to add more props to the small variant we can do so with a single prop. Would also do `small | medium` for sizes

Reply:
I agree with the name of the sizes. Not so sure about changing the name to `variant`, I get your point to generalize in case there will be future changes but those changes are not planned at all and even that property wasn't really supposed to be. I'm just thinking that having variant=small would create more expectations then just to have the padding changed. while consuming I would expect more to happen.

Reply:
I see your point. But: 

>  those changes are not planned at all

> even that property wasn't really supposed to be.

Do you see my point? We will never know what will and won't be needed, that's why I suggested a variant. Also, I don't think it's a matter of expectations, there should be a design for the different variants and it's up to the designers to use them correctly in figma and it's up to the devs to consume them correctly. 

Having said that,I don't want to block this, so feel free to merge 😄 👌